string text
string lang
bool force_stream_mode